src/main/java/net/openhft/chronicle/queue/ChronicleHistoryReaderMain.java

```diff
@@ -105,21 +159,26 @@ protected void printHelpAndExit(final Options options, int status, String messag
                 null,
                 true
         );
-        writer.flush();
-        System.exit(status);
+        writer.flush(); // Ensure everything is printed
+        System.exit(status); // Exit with provided status
```

Comment:
Consider removing these single line javadocs - the code in these instances describes what it is doing - doubling up with the comment is not needed